suites/portal/discoveryPageTest.js

```diff
@@ -30,8 +30,10 @@ Scenario('Publish a study, search and export to workspace @requires-indexd @requ
   mds, users, discovery, files, indexd, home, workspace,
 }) => {
   // dynamically get UID field name from portal config
-  const UIDFieldName = await bash.runCommand('gen3 secrets decode portal-config gitops.json | jq \'.discoveryConfig.minimalFieldMapping.uid\'').replace(/^"(.*)"$/, '$1');
+  const UIDFieldName = bash.runCommand('gen3 secrets decode portal-config gitops.json | jq \'.discoveryConfig.minimalFieldMapping.uid\'').replace(/^"(.*)"$/, '$1');
   expect(UIDFieldName).to.be.a('string').that.is.not.empty;
+  const studyPreviewField = bash.runCommand('gen3 secrets decode portal-config gitops.json | jq \'.discoveryConfig.studyPreviewField.field\'').replace(/^"(.*)"$/, '$1');
+  expect(studyPreviewField).to.be.a('string').that.is.not.empty;
```

Comment:
This field is not mandatory, so it is legal if a discovery page doesn't want to show this sumamry row

Reply:
hmm ... does that mean we should remove the validation to search through summary?